app/components/faq/QuestionList.component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Grid, List, Header, Menu, Icon } from 'semantic-ui-react';
+import './style.scss';
+import { rightArrowIcon } from "./images";
+
+const QuestionList = (props) => (
+  <React.Fragment>
+    <Grid stackable centered > 
+      <Grid.Column width={8} style={{ paddingBottom: '3rem' }}>
+        <List>
```

Comment:
Do you need this `<List/>` wrapper? The content is already wrapped in a fragment.